index.html

```diff
@@ -245,6 +245,12 @@
               show icons in chart
             </label>
           </div>
+          <div class="form-check">
+            <input class="form-check-input" type="checkbox" id="relative_azerite_forge" style="cursor: pointer">
+            <label class="form-check-label" style="cursor: pointer" id="translate_relative_azeriteforge" for="relative_azerite_forge">
+              relative azerite import
```

Comment:
Please change this to "import <u>relative</u> AzeriteForge Trait stacking import". Users should know what exactly will change, even though this change makes the text longer and the settings pop-up more ugly.  :< If you have a different suggestion for this string, let me know.

Reply:
"Import relative AzeriteForge Trait stacking" maybe?

Reply:
Ah, mouseover is a good idea, in that case I could see a "import relative AzeriteForge" label as sufficient if that gets a tooltip (below?) with an explanation

Reply:
Maybe like this?
![image](https://user-images.githubusercontent.com/28957605/64487795-3fff9d80-d23f-11e9-83ca-a02ebd239120.png)

Reply:
Great explanation, just use the tooltips of bootstrap for that.

https://getbootstrap.com/docs/4.3/components/tooltips/#examples

Reply:
![image](https://user-images.githubusercontent.com/28957605/64487856-1a26c880-d240-11e9-9652-4bbea180bfa0.png)

Yea looks a lot better haha

Reply:
Great, I'll pull once again